shared/chat/conversation/banner.desktop.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+import React from 'react'
+import {Box, Icon, Text, Header as CommonHeader} from '../../common-adapters'
+import {globalStyles, globalColors, globalMargins} from '../../styles'
+
+import type {Props, ErrorVariant, InviteVariant} from './banner'
+
+const ErrorBanner = (props: ErrorVariant) => {
+  return (
+    <CommonHeader style={{...globalStyles.flexBoxColumn, backgroundColor: globalColors.red}} type='Strong'>
```

Comment:
We should likely remove the windowdragging stuff from commonheader and make that a prop